<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="0" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="750" started-at="2020-02-25T20:36:48Z" finished-at="2020-02-25T20:36:49Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:walmart.Runner.Runner@4f7d0008]" name="feature" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="750" started-at="2020-02-25T20:36:48Z" finished-at="2020-02-25T20:36:49Z">
      <class name="walmart.Runner.Runner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:walmart.Runner.Runner@4f7d0008]" name="setUpClass" is-config="true" duration-ms="470" started-at="2020-02-25T15:36:48Z" finished-at="2020-02-25T15:36:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="writeExtentReport()[pri:0, instance:walmart.Runner.Runner@4f7d0008]" name="writeExtentReport" is-config="true" duration-ms="0" started-at="2020-02-25T15:36:49Z" finished-at="2020-02-25T15:36:49Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.cucumber.listener.Reporter.loadXMLConfig(Reporter.java:66)
	at walmart.Runner.Runner.writeExtentReport(Runner.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:225)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:114)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- writeExtentReport -->
        <test-method status="SKIP" signature="tearDownClass()[pri:0, instance:walmart.Runner.Runner@4f7d0008]" name="tearDownClass" is-config="true" duration-ms="0" started-at="2020-02-25T15:36:49Z" finished-at="2020-02-25T15:36:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- walmart.Runner.Runner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
